packages/functions/src/optionSchemas.ts

```diff
@@ -9,6 +9,7 @@ export const optionSchemas: Record<string, CustomFunctionOptionsSchema> = {
     properties: {
       keyedBy: {
         type: 'string',
+        description: 'key to sort an object by',
```

Comment:
Is there a specific reason why everything here starts from a small letter? 🤔 

If this is something we want to reflect in UI later, it would be better to have it in a "sentence" format, ie starting with big letter and ending with a dot?

Reply:
Could be yeah. I mostly copied it from docs and didn't change anything.